riggerlib/client.py

```diff
@@ -21,6 +21,10 @@ def _very_lazy_pirate_request(socket, data):
     try:
         socket.send_json(data)
         socks = dict(poll.poll(REQUEST_TIMEOUT))
+        if not socks:
+            # This seem to happen when the socket is not connected.
+            raise RuntimeError(f'No result from polling the {poll}.')
```

Comment:
this one breaks on old python, do we want to drop support and shift to 3.6+ )whould need a setup/travis config update as well)

Reply:
This should be fixed by now.

Reply:
I'm happy to do that if CFME is

Reply:
@psav @RonnyPfannschmidt  I see only python 3.7 installed on our PRT.

Reply:
I'm definitely okay with dropping py2.7 support here.

CFME moved to 3.7, 3.6+ support should work just fine for us.